includes/3rd-party/themes/divi.php

```diff
@@ -89,4 +89,54 @@ function sensei_fix_divi_learning_mode_video_template_excerpt() {
 		remove_filter( 'render_block_core/post-excerpt', array( ET_GB_Block_Post_Excerpt::instance(), 'render_block' ) );
 	}
 }
+
+/**
+ * Enqueue Course theme-specific Learning Mode styles.
+ */
+function sensei_load_learning_mode_style_for_divi_theme() {
+	$course_id       = Sensei_Utils::get_current_course();
+	$is_course_theme = 'divi' === wp_get_theme()->get_template();
```

Comment:
Should this variable be named `$is_divi_theme` or something like that?

Reply:
Fixed  here 2703ebb